.gitignore

```diff
@@ -17,6 +17,9 @@
 mono_crash.*
 
 # Build results
+[Oo]bj
+[Bb]in
```

Comment:
hm, I am still seeing files under bin/ and obj/ folder in this PR that shouldn't be here. Could you please exclude them from the github?